assets/xml/objects/object_tanron4.xml

```diff
@@ -1,14 +1,18 @@
 ﻿<Root>
     <File Name="object_tanron4" Segment="6">
```

Comment:
We want to have a comment above the `File` node in the XML describing what this object is. You could say something like "Assets for seagulls" or something like that.